sphinx/extension.py

```diff
@@ -67,10 +72,14 @@ def verify_needs_extensions(app: Sphinx, config: Config) -> None:
                     fulfilled = False
 
         if not fulfilled:
-            raise VersionRequirementError(__('This project needs the extension %s at least in '
-                                             'version %s and therefore cannot be built with '
-                                             'the loaded version (%s).') %
-                                          (extname, reqversion, extension.version))
+            raise VersionRequirementError(
+                __(
+                    'This project needs the extension %s at least in '
+                    'version %s and therefore cannot be built with '
+                    'the loaded version (%s).'
+                )
+                % (extname, reqversion, extension.version)
+            )
```

Comment:
```suggestion
            raise VersionRequirementError(__(
	            'This project needs the extension %s at least in '
	            'version %s and therefore cannot be built with '
	            'the loaded version (%s).'
			) % (extname, reqversion, extension.version))
```

If possible, I would have appreciated this kind of format. Note that ruff sometimes allows '%' and ')' to be on the same line, but I'm not sure if'ts possible to have this kind of compact writing. Honestly, having too many new lines (especially '%' not on the same line) does not help me for reading the code itself (the current version is worse, the new ruff is better but '%' on a new line is not good and my version might be too compact.

I know I am nitpicking with those PR's but if we are using the same format, I prefer having a format that does not make the code readibility worse.

Reply:
this is not possible to configure

Reply:
The lack of configuration is... kind of the point

In this particular case, if you wanted to change this behaviour you'd have to open an issue in Black, since ruff is matching the upstream format style